src/app/pages/friends-page/friends-page.component.ts

```diff
@@ -30,10 +30,10 @@ export class FriendsPageComponent implements OnInit {
     ngOnInit(): void {
         this.profilServices.getFriends().subscribe({
             next: (users: any) => {
-                if (users && users.idFriends) {
-                    this.users = users.idFriends;
+                if (users && users.friends) {
+                    this.users = users.friends;
                     const user = JSON.parse(localStorage.getItem('user') as string);
-                    user.friends = users.idFriends;
+                    user.friends = users.friends;
```

Comment:
pk t'as besoin de faire ca ?

Reply:
Parce qu'avant comme les amis c'était une collection j'ajoutais les amis dans un tableau sur l'objet user du front mais au final comme y'en a plus besoin je l'ai suppr

Reply:
en fait je vois pas pk tu stocke les amis dans le user

Reply:
pour moi a chaque fois que tu charge la page tu recup les amis

Reply:
Bah la du coup maintenant je le fais plus avant c'était juste une question pratique pour la suite, maintenant je récupère juste la liste des amis avec getFriends pour être sur de ta liste d'ami dans la db, après je la stocke dans la variable pour le composant card mais c'est tout

Reply:
en fait je comprends pas pk t'as 
this.users = users.friends;
et 
const user = JSON.parse(localStorage.getItem('user') as string);
user.friends = users.friends;

Reply:
tu devrais juste avoir this.friends = res.friends